src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
I don't understand the logic for deciding that an object belongs to only one dataset... If, say, you wish to specify which individuals you did your GWAS on, what object would you use? A dataset no? And we definitely would want individuals to be re-used in multiple cohorts.